apps/docs/content/guides/platform/advanced-log-filtering.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'Advanced Log Filtering'
+description: 'Filter logs with regular expressions'
+---
+
+# Querying the logs
+
+## Understanding field references
+
+The log tables are queried with subset of BigQuery SQL syntax. They all have three columns: `event_message`, `timestamp`, and `metadata`.
+
+| column        | description                 |
+| ------------- | --------------------------- |
+| timestamp     | time event was recorded     |
+| event_message | the log's message           |
+| metadata      | information about the event |
+
+The `metadata` column is an array of JSON objects that stores important details about each recorded event. For example, in the postgres table, the `metadata.parsed.error_severity` field indicates the error level of an event. To work with its values, you need to `unnest` them using a `cross join`.
+
+This approach is commonly used with JSON and array columns, so it might look a bit unfamiliar if you're not used to working with these data types.
+
+```sql
+select
+  event_message,
+  error_severity, -- found in parsed
+  user_name -- found in parsed
+from
+  postgres_logs
+  -- extract first layer
+  cross join unnest(postgres_logs.metadata) as metadata
+  -- extract second layer
+  cross join unnest(metadata.parsed) as parsed;
+```
+
+## Expanding results
+
+Logs returned by queries may be difficult to read in table format. A row can be double-clicked to expand the results into more readable JSON:
+
+![Expanding log results](/docs/img/guides/platform/expanded-log-results.png)
+
+## Filtering with [regular expressions](https://en.wikipedia.org/wiki/Regular_expression)
+
+The Logs use BigQuery Style regular expressions with the [regexp_contains function](https://cloud.google.com/bigquery/docs/reference/standard-sql/string_functions#regexp_contains). In its most basic form, it will check if a string is present in a specified column.
+
+```sql
+select
+  cast(timestamp as datetime) as timestamp,
+  event_message,
+  metadata
+from postgres_logs
+where regexp_contains(event_message, 'is present');
+```
+
+There are multiple filters that you should consider using:
+
+### Find messages that start with a phrase
```

Comment:
These could be summarised and condensed to a table, as these are basic explainers of regexp operators

Reply:
@charislam, should we change it to tabular form?

Reply:
I think the the table form is easier to scan

Reply:
lets get this in first